src/main/java/net/greenfieldmc/core/utilities/services/UtilityCommands.java

```diff
@@ -48,6 +48,32 @@ public void tryEnable(Plugin plugin, Module module) throws Exception {
                         player.teleport(new Location(player.getLocation().getWorld(), player.getLocation().getX(), -20000000, player.getLocation().getZ()));
                     }, 54);
                 }).register(plugin);
+
+        CommandBuilder.of("isprime")
+                .permission("greenfieldcore.isprime")
+                .description("Check if a number is prime.")
+                .then("number", PdkArgumentTypes.integer())
+                .executes(ctx -> {
+                    int number = ctx.getTyped("number", Integer.class);
+                    if (number <= 1) {
+                        ctx.asPlayer().sendMessage("§c" + number + " is not prime (must be greater than 1).");
+                        return;
+                    }
```

Comment:
Once the first comment in this file is resolved, this if statement checking if the number is <= 1 can be removed entirely.